### ISSUE TITLE
A little advice from a foreign friend

### ISSUE THREAD

**Issue by myh66** [DeepLcom/openapi#5]
You can check out the translation model of this plugin, it would be simply awesome if the officials would collaborate with them or make a similar plugin
His translation is based on the original with annotations
https://twitter.com/immersivetran
https://microsoftedge.microsoft.com/addons/detail/%E6%B2%89%E6%B5%B8%E5%BC%8F%E7%BF%BB%E8%AF%91/amkbmndfnliijdhojkpoglbnaaahippg
![image](https://github.com/DeepLcom/openapi/assets/53999290/a45f11e2-ea78-4208-8c80-215c21b0f8ad)
This translation model is great for students who are learning a foreign language!

**Comment by JanEbbing** [comment]
Hi, this plugin looks interesting but we don't own it (and it doesn't look like it's open source? It has a [GitHub](https://github.com/immersive-translate/immersive-translate) but it looks like it's already transpiled TypeScript. So the developer who made it would need to integrate our API.

**Comment by myh66** [comment]
Yes, it's possible to use your API on his plugin, I'm struggling with the fact that our country doesn't support the opening of deeply API pro, this is great. I'm using Mastercard and Express and it doesn't work, it looks like it only supports a few countries that are written on the website, hope,fully it will come to the Chinese market soon.

---Original---
From: ***@***.***&gt;
Date: Tue, Sep 5, 2023 15:58 PM
To: ***@***.***&gt;;
Cc: "Evan ***@***.******@***.***&gt;;
Subject: Re: [DeepLcom/openapi] A little advice from a foreign friend (Issue#5)




 
Hi, this plugin looks interesting but we don't own it (and it doesn't look like it's open source? It has a GitHub but it looks like it's already transpiled TypeScript. So the developer who made it would need to integrate our API.
